### PR TITLE
Prepare for alpha release

### PR DESCRIPTION
**Pull request by bergundy**
- Update README
- Minor API reference changes

### CODE REVIEW

**Review by sw-yx — APPROVED** [review]
LGTM. lets :shipit:

File: packages/worker/src/worker.ts [hunk 1]
```diff
@@ -165,7 +165,7 @@ export interface WorkerOptions {
   /**
    * Maximum number of Activities to execute concurrently.
    * Adjust this to improve Worker resource consumption.
-   * @default 200
+   * @default 10
```
Comment (Sushisource) [inline packages/worker/src/worker.ts:168]: :sunglasses:

File: packages/worker/src/worker.ts [hunk 2]
```diff
@@ -176,10 +176,10 @@ export interface WorkerOptions {
   maxConcurrentWorkflowTaskExecutions?: number;
```
Comment (Sushisource) [inline packages/worker/src/worker.ts:176]: Should this still be so high? Might make sense to crank the default down here a bit too.

File: packages/worker/src/worker.ts (hunk 1 above)
Comment (sw-yx) [reply packages/worker/src/worker.ts:168]: wait why is this 😎 i thought it would be 😢

Comment (Sushisource) [reply packages/worker/src/worker.ts:168]: It's a sardonic :sunglasses:, lol

Comment (bergundy) [reply packages/worker/src/worker.ts:168]: It bumped it up to 100, I got it wrong, it's 10 WFTs and 100 ATs.
After we merge the bundler branch I'll bump it back up to 100.

File: packages/worker/src/worker.ts (hunk 2 above)
Comment (bergundy) [reply packages/worker/src/worker.ts:176]: See comment above, thanks for pointing this out.

**Pull request merged**